src/CookBlog.Data/Exceptions/InvalidFileExtensionsException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using CookBlog.Api.Core.Exceptions;
+
+namespace CookBlog.Core.Exceptions;
+
+public sealed class InvalidFileExtensionsException : CustomException
+{
+    public string File { get; }
+    
+    public InvalidFileExtensionsException(string file) : base($"File:'{file}' is invalid image")
```

Comment:
ta wiadomośc to jest taka sama jak File:'{file}' is invalid dodaj że rozszerzenie złe